docker-compose.yml

```diff
@@ -24,8 +24,12 @@ boulder:
       - 9380:9380 # mail-test-srv
       - 9381:9381 # mail-test-srv
     links:
+      - bhsm:boulder-hsm
       - bmysql:boulder-mysql
       - brabbitmq:boulder-rabbitmq
+bhsm:
+    build: test/
```

Comment:
Actually, I think this setup means that each Travis build has to build the Dockerfile-hsm image, which includes a bunch of fetching and building. Can we make this a fetchable image a la boulder-tools? Or perhaps, to simplify things, we could incorporate the pkcs11-proxy stuff into boulder-tools, and run the boulder-tools image with a different command to serve as the remote end of the pkcs11 proxy?